skymarshal/dexserver/web/themes/concourse/styles.css

```diff
@@ -96,44 +114,34 @@
   background-color: #196AC8;
 }
 
-.theme-form-row {
+.theme-form-row:not(:last-child) {
   display: block;
-  margin:  0 auto 15px;
+  margin:  0 auto 16px;
 }
 
 .theme-form-input {
-  border: 1px solid #6D6D6D;
+  border: 1px solid #A4A4A4;
   display: block;
-  width: 230px;
-  height: 38px;
-  margin: 0 auto;
-  padding: 6px 12px;
+  width: 390px;
+  height: 52px;
+  padding: 16px;
   background-color: #1E1D1D;
-  font-size: 16px;
-}
-
-.theme-form-input:focus,
-.theme-form-input:active {
-  border-color: white;
+  font-size: 20px;
```

Comment:
What was the reason for increasing the font size? The difference in font size between most other text (12px) and the login screen is pretty jarring.

Reply:
https://www.figma.com/file/fuoFSVcTGptiS21P2POA2XbN/Concourse-UI-kit-and-Experiments?node-id=2590%3A170427

it is based on the sketch. I am not sure 100% but it might want to align with the font size used in the button?

Reply:
Sorry @xtremerui - that might be my fault. I just made a new example based on a screenshot, but the screenshot was made on a retina monitor, which might have skewed the font sizes.

It's probably better to reduce the font size back down to its original scale.